tests/Wallabag/CoreBundle/Command/InstallCommandTest.php

```diff
@@ -201,6 +202,16 @@ public function testRunInstallCommandChooseResetSchema()
 
     public function testRunInstallCommandChooseNothing()
     {
+        /*
+         *  [PHPUnit\Framework\Error\Warning (2)]
+         *  filemtime(): stat failed for /home/runner/work/wallabag/wallabag/var/cache/tes_/ContainerNVNxA24/appAppKernelTestDebugContainer.php
+         *
+         * I don't know from where the "/tes_/" come from, it should be "/test/" instead ...
```

Comment:
I often have this kind of folder with `de_` instead of `dev`.

Reply:
On which Symfony version?

Reply:
on the current one. For example, today on wallabag.it (it's with a `~` but same issue I think) 

<img width="459" alt="Capture d’écran 2022-11-24 à 11 47 59" src="https://user-images.githubusercontent.com/121870/203765445-9d3a8c3e-c916-4311-8baa-e327918353c3.png">